pulsar-broker/src/test/java/org/apache/pulsar/broker/auth/MockedPulsarServiceBaseTest.java

```diff
@@ -304,11 +307,19 @@ protected void stopBroker() throws Exception {
         // set shutdown timeout to 0 for forceful shutdown
         pulsar.getConfiguration().setBrokerShutdownTimeoutMs(0L);
         pulsar.close();
+        cleanupMock(pulsar);
         pulsar = null;
         // Simulate cleanup of ephemeral nodes
         //mockZooKeeper.delete("/loadbalance/brokers/localhost:" + pulsar.getConfiguration().getWebServicePort(), -1);
     }
 
+    protected void cleanupMock(Object mock) {
```

Comment:
what about adding this method to `BrokerTestUtil` ?

Reply:
I'm simplifying this. I'll push an update soon. It's sufficient to call Mockito.reset

Reply:
Changes pushed